### PR TITLE
Request for individual verification

### PR DESCRIPTION
**Pull request by richtier**
relies on [this PR](https://github.com/uktrade/directory-api-client/pull/93)

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] Requirements have been compiled.
 - [x] Change is accessible to the standards we subscribe to.
 - [x] Add a printscreen to the PR

image

image

image

### CODE REVIEW

File: profile/fab/forms.py [hunk 1]
```diff
@@ -536,3 +536,7 @@ def clean_expertise_products_services(self):
         return tokenize_keywords(
             self.cleaned_data['expertise_products_services']
         )
+
+
+class IdentityVerificationRequetForm(forms.Form):
```
Comment (eadpearce) [inline profile/fab/forms.py]: "Requet"?

File: profile/fab/views.py [hunk 2]
```diff
@@ -420,3 +420,20 @@ class PersonalDetailsFormView(
     def form_valid(self, form):
         self.create_user_profile(form)
         return super().form_valid(form)
+
+
+class IdentityVerificationRequetFormView(SuccessMessageMixin, FormView):
```
Comment (eadpearce) [inline profile/fab/views.py]: hey at least you are consistent in your misspellings

File: profile/templates/fab/profile.html [hunk 3]
```diff
@@ -50,9 +50,14 @@
     {% if not company.email_address or not company.description or not company.is_verified or not company.is_published %}
     <div class="width-half">
         <h3 class="heading-medium">Next steps</h3>
-        <p>Your business profile has not been published. Complete the steps below to promote your business to overseas buyers</p>
+        {% if company.is_identity_check_message_sent %}
+            <p>We will be in contact within the next XX days and have sent you a confirmation email to {{ request.user.email }}</p>
```
Comment (eadpearce) [inline profile/templates/fab/profile.html]: assuming "XX" will be replaced with number before it's due to go live?

File: profile/fab/views.py (hunk 2 above)
Comment (richtier) [reply profile/fab/views.py]: [maybe I'm french](https://www.urbandictionary.com/define.php?term=requet)

File: profile/templates/fab/profile.html (hunk 3 above)
Comment (richtier) [reply profile/templates/fab/profile.html]: yep. this is effectively feature flagged insofar as the `is_identity_check_message_sent` will only be `True` if the message has been sent and that will only be available to folks that meet this criterion:
 - are non-companies house companies
 - are not verified

of which there are none until we allow non-companies house companies to enrol :)

**Pull request merged**